event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/CristinMapper.java

```diff
@@ -289,13 +295,43 @@ private static BigDecimal extractAuthorPointsForAffiliation(ScientificPerson sci
         return new BigDecimal(scientificPerson.getAuthorPointsForAffiliation());
     }
 
-    private static URI getTopLevelOrganization(ScientificPerson scientificPerson, List<CristinLocale> institutions) {
+    private URI getTopLevelOrganization(ScientificPerson scientificPerson, List<CristinLocale> institutions) {
+        var list = institutions.stream()
```

Comment:
Rename var to something more describing than `list`?

Reply:
Maybe findFirst() is what you want here?

Reply:
Refactoring to:

```
    private URI getTopLevelOrganization(ScientificPerson scientificPerson, List<CristinLocale> institutions) {
        return institutions.stream()
                             .filter(cristinLocale -> hasSameInstitutionIdentifier(scientificPerson, cristinLocale))
                             .map(CristinMapper::constructInstitutionId)
                             .findFirst().orElseGet(() -> extractInstitutionIdentifierForTransferredInstitution(scientificPerson, institutions));
    }
```